libs/domain/search/src/renderers/value-renderer/rating.renderer.ts

```diff
@@ -14,12 +14,11 @@ export const ratingFacetRenderer = {
         <oryx-search-facet-rating
           @oryx.select=${selectListener}
           .name=${facet.name}
-          .renderLimit=${options.renderLimit}
           .open=${options.open}
           ?disableClear="${!options.enableClear}"
-          .multi=${facet.type === FacetType.Multi}
-        >
-        </oryx-search-facet-rating>
+          min="1"
+          max="4"
```

Comment:
how come these are hardcoded?

Reply:
I dropped default values for these props in component (to decrease amount of code and increase flexibility), facet's min and max are used by default, if values are not provided. The values are hardcoded in the renderer due to we don't have mechanism to pass per-component options on EB side for facet-navigation component, only the common set. 

For now the only way to override this hardcoded min and max - to override the renderer for the rating. But I agree, our current implementation of the facets at all are so limited

Reply:
That's a bummer. In the former implementation, we could override the default values in the application configuration. The lack of EB is not so relevant as it's not a product that we ship anytime soon. Can we bring it back?

Reply:
The idea was to allow customers to either come up with min=4 and max=4 (only showing 4-star, which is quit common) or allow to include 5